src/openrct2/world/Banner.h

```diff
@@ -22,22 +22,28 @@
 
 #define BANNER_NULL 255
 #define MAX_BANNERS 250
-#define BANNER_INDEX_NULL -1
+#define BANNER_INDEX_NULL ((uint8)-1)
```

Comment:
What's the difference between `BANNER_NULL` and `BANNER_INDEX_NULL`?

Reply:
`BANNER_NULL` is used for slots in `rct_banner gBanners[]` that are not in use.
`BANNER_INDEX_NULL` is much like `SPRITE_INDEX_NULL`, used by tile elements and sprites to refer to an entry in the banner list.